central/sensor/service/connection/connection_impl.go

```diff
@@ -280,6 +280,8 @@ func (c *sensorConnection) processComplianceResponse(ctx context.Context, msg *c
 	switch m := msg.Response.(type) {
 	case *central.ComplianceResponse_ApplyComplianceScanConfigResponse_:
 		return c.complianceOperatorMgr.HandleScanRequestResponse(ctx, m.ApplyComplianceScanConfigResponse.GetId(), c.clusterID, m.ApplyComplianceScanConfigResponse.GetError())
+	case *central.ComplianceResponse_DeleteComplianceScanConfigResponse_:
+		return c.complianceOperatorMgr.HandleDeleteScanRequestResponse(ctx, m.DeleteComplianceScanConfigResponse.GetId(), c.clusterID, m.DeleteComplianceScanConfigResponse.GetError())
```

Comment:
Does the function this is calling still exist?  Should it?  might be worthwhile to receive these responses and log the errors as warnings if we get them.